exercises/poker/examples/success-standard/src/Poker.hs

```diff
@@ -0,0 +1,51 @@
+module Poker (bestHands) where
+
+import Data.Maybe --(fromJust)
```

Comment:
I'd like to be able to keep track of where identifiers are coming from, so I'd say to uncomment the lists!

Reply:
Forgot about that